### PR TITLE
Add "saving" and "clear" functionality

### PR DESCRIPTION
**Pull request by ArthurGamerHD**
Add saving functionality to allow know items to be saved on the Storage field of the programable block, to avoid "losing" data about an item if the PB is recompiled while theres no sample of this item in any cargo container

also added "clear" functionality to allow removing items with the amount equal to 0 from the storage dictionary

### CODE REVIEW

**Comment by Brianetta** [comment]
This looks very good, and saves me writing this feature myself, thanks! I'll test it in-game tomorrow, then very likely merge it.

**Comment by ArthurGamerHD** [comment]
> Owner

Sure, xD

You're welcome,
This script is very useful, helping you also helps me, as I use it a lot.

just adding a thing i misses from it, especially in servers, since on each reboot, it forgets the missing items.

and also fixing a bug to allow usage on sideways Text Panel, that i really LOVE the looks.
image

I've wanted to do these things for a long time, but I never found the time to do them, so I'm taking advantage of the 3-day national holiday, +2-day of the weekend, to do some pending things in my scripts (and third-party scripts)

File: All My Stuff/MDK/MDK.paths.props
Comment (ArthurGamerHD) [inline All My Stuff/MDK/MDK.paths.props:1]: Ignore, reverted on next commit

File: All My Stuff/All My Stuff.csproj
Comment (ArthurGamerHD) [inline All My Stuff/All My Stuff.csproj:1]: Ignore, reverted on next commit

File: All My Stuff_Backup_2024-02-11-174429311.zip
Comment (ArthurGamerHD) [inline All My Stuff_Backup_2024-02-11-174429311.zip:1]: Ignore, reverted on next commit

**Pull request merged**